.env.example

```diff
@@ -2,9 +2,12 @@
 # development setup #
 #####################
 
+# These are needed for docker-compose
+# Thus we need to specify --env-file .env.development when calling docker-compose
 REACT_PORT=3000  # local port for frontend
 NODE_PORT=3001  # local port for backend
 SERVER_NAME=yourdomainename.com  # your domain name, for the nginx configuration
+MODE=dev|prod # you must chose !
```

Comment:
what happens if the developer tries to use something not in `dev|prod`?

Reply:
It will simply not found the associated template and crash, I must confess I don't know if the error's nice, but in my defense this is configuration stuff, you don't expect a user to writes nonsense here (like for example qtile config crashes and load default in case of semantics error in the config file).

Reply:
just asking, a crash is far better than unexpected behaviour :relieved:

Reply:
I unresolved this one as a reminder to test this behavior once the dev env works!
This shouldn't be a problem but better be sure :wink:

Reply:
didn't test it yet

Reply:
tks for raising the issue! 
Actually it wasn't working so the website wasn't up on port 80 but teh container wasn't crashing, quite hard to understand the logs. So I armored the script against that: [fc1c601](https://github.com/iScsc/iscsc.fr/pull/86/commits/fc1c601fcb86a91b356310bdaab8fc888a9a9f08) tks again!

Reply:
Here is what happen is you put `dev|prod` with fc1c601
```
[atxr@arch iscsc.fr]$ docker logs iscscfr-nginx-1 
nginx template: 'nginx.conf.dev|prod.template' not found, exiting
```
@amtoine I'll let you close this